index.html

```diff
@@ -0,0 +1,189 @@
+                </div>
+            </div>
+            
+            </section>
+
+
+            <section>
+    <div class="consuling">
+
+        </div>
+    <footer class="footer">
+        <div class="footer_header">
+            <div class="logo_footer">
+                <p class="logo_footer_l">KMarketer</p>
+            </div>
+            <div class="social_info">
+            <nav class="nav_bar_foot">
+                <ul class="social_ul">
+                    <li><img src="img/facebook.svg"></li>
+                    <li><img src="img/Instagram.svg"></li>
+                    <li><img src="img/youtube.svg"></li>
+                    <LI><img src="img/twitter.svg"></LI>
+                </ul>
+                </nav>
+            </div>
+        </div>
+        <div class="divider">
+        </div>
+        <hr class="footer_divider">
+        <div class="footer_info">
+            <nav class="footer_nav">
+                <div class="nav_block">
+                    <h4>Company info</h4>
+                    <ul>
+                        <li><a href="#"> About us</a></li>
+                        <li><a href="#"> Carrier</a></li>
+                        <li><a href="#">We are hiring</a></li>
+                        <li><a href="#"> Blog</a></li>
+                    </ul>
+                </div>
+                <div class="nav_block">
+                    <h4>Legal</h4>
+                    <ul>
+                        <li><a href="#"> About Us</a></li>
+                        <li><a href="#">Carrier</a></li>
+                        <li><a href="#"> We are hiring</a></li>
+                        <li><a href="#"> Blog</a></li>
+                    </ul>
+                </div>
+                <div class="nav_block">
+                    <h4>Features</h4>
+                    <ul>
+                        <li><a href="#"> Business and Marketing</a></li>
+                        <li><a href="#"> User analitic</a></li>
+                        <li><a href="#"> Live chat</a></li>
+                        <li><a href="#"> Unlimited support</a></li>
+                    </ul>
+                </div>
+                <div class="nav_block">
+                    <h4>Company info</h4>
+                    <ul>
+                        <li><a href="#"> Android and Ios</a></li>
+                        <li><a href="#"> Watch demo</a></li>
+                        <li><a href="#"> Customers</a></li>
+                       <li><a href="#"> API</a></li> 
+                    </ul>
+                </div>
+            </nav>
+            <div class="subscribe_info">
+                <h4>Get in touch</h4>
+            <div class="input_subscribe">
+                <input class="form_input" type="email">
+                <input class="form_submit" type="submit">
+
+
+            </div>
+
+            </div>
+
+        </div>
+        <div class="rights">
+            <p class="love_figma">Made With Love By Figmaland All Right Reserved </p>
+
+
```

Comment:
Please align the code in a proper way. It is really difficult to review when there is a mess with tags and spaces